tests/js/testUtils.js

```diff
@@ -56,4 +56,12 @@ https://github.com/gpii/universal/LICENSE.txt
         });
     };
 
+    gpii.tests.utils.simulateKeyEvent = function (onElement, keyEvent, keyCode) {
```

Comment:
I've renamed one function to triggerKeyEvent(). This two functions are not exchangable because current triggerKeyEvent() calles on jQuery triggerHandler(). According to http://api.jquery.com/triggerhandler/, this function executes all handlers bound with jQuery for the event type but will not call .event() on the element it is triggered on. While the other simulateKeyEvent() uses javascript dispatchEvent() function to dispatch an event at the specified target.